src/RealtimeClient.ts

```diff
@@ -160,6 +167,14 @@ export default class RealtimeClient {
     }, this.reconnectAfterMs)
 
     this.fetch = this._resolveFetch(options?.fetch)
+    if (options?.worker) {
```

Comment:
let's make this as simple as possible so the dev doesn't even have to think about this. we can do a check to see if the current env is browser and if so use the webworker.

if we really wanted to improve that we can check page visibility api to see if tab is active or not and then activate webworker.

Reply:
Most of the worker stuff is actually like this according to the issue comments so I would rather keep it like this and avoid creating a new layer of issues